backend/products/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+
+# Create your models here.
+
+class Product(models.Model):
+    name = models.CharField(max_length=200)
+    description = models.TextField(max_length=600, blank=True)
+    category = models.CharField(max_length=100)
```

Comment:
shouldn't we add choices here?

Reply:
Can you please elaborate? I didnt choices as what?

Reply:
By choices i mean should we make it an enum (sort of), [reference](https://docs.djangoproject.com/en/4.2/ref/models/fields/#choices)